src/x509_helper.cc

```diff
@@ -178,17 +184,43 @@ int main() {
            "x509 authz helper invoked, connected to cvmfs process %d",
            getppid());
 
+  CheckSciToken_t checker = NULL;
+  if (strcmp(basename(argv[0]), "cvmfs_scitoken_helper") == 0) {
+    checker = SciTokenLib::GetInstance();
+  }
+
   while (true) {
     msg = ReadMsg();
     LogAuthz(kLogAuthzDebug, "got authz request %s", msg.c_str());
     AuthzRequest request = ParseRequest(msg);
     string proxy;
+    LogAuthz(kLogAuthzDebug, "Executable: %s", basename(argv[0]));
```

Comment:
Hoist this debug message outside the loop so we only get it once per invocation.